### PR TITLE
[UI] improvement on menu 

### LINKED ISSUE #739: Improvement: Menu Styling
**Current State of the Meshery.io site**
The current navigation menu is less than attractive. Screenshot below:
Screen Shot 2022-06-14 at 1 54 20 PM

**Desired State of the Meshery.io site**
The navigation menu might be prettified with a bit of rounded corners and shadow or something similar. This issue is to request a contributor to brainstorm and propose an improvement.

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Rohith-Raju**
Signed-off-by: Rohith Raju [rohithraju488@gmail.com](mailto:rohithraju488@gmail.com)

**Description**
Added icons and redesigned Menu Styling.

This PR fixes #739 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Rohith-Raju** [comment]
@leecalcote @warunicorn19 how are the icons looking? any changes ? and  I need to come up with catalog and service mesh icons.

**Review by asubedy — CHANGES_REQUESTED** [review]
image
image

1. A different Icon for the contribution would look good this is looking kinda weird 😅 
2. The Layer5 icon is blurred
3. Please use the official Discuss logo, since we are using the official one for the others.

**Comment by leecalcote** [comment]
Still has "do-not-merge" label...

**Comment by Rohith-Raju** [comment]
@leecalcote, still need to decide on some icons. catalog and service mesh

**Comment by leecalcote** [comment]
@Rohith-Raju please recolor the svgs to use the same color as the menu items (hyperlinks).

**Comment by leecalcote** [comment]
Service mesh icon - https://layer5.io/static/cb310234b6631abcabb632a85974a3dd/service-mesh.svg

**Comment by leecalcote** [comment]
@Rohith-Raju let's be consistent with either fill or outline style icons. Email, Blog, and Programs need a fill-style icon.

**Comment by Rohith-Raju** [comment]
@leecalcote Sure, I'll try either of them and then come back for feedback

**Comment by leecalcote** [comment]
Merge conflict...

**Review by leecalcote — CHANGES_REQUESTED** [review]
Looking pretty good. Will you filter() the color of the SVG icons to match that of the charcoal color used for the menu item hyperlinks?

**Comment by Rohith-Raju** [comment]
Sure @leecalcote

**Comment by Rohith-Raju** [comment]
@leecalcote 
image
image

**Comment by leecalcote** [comment]
We’re leaving out icons in mobile view intentionally, right?

File: _data/navigation.yml [hunk 1]
```diff
@@ -25,54 +25,71 @@
...
     - name: Youtube
       link: https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1
       new_window: true
       external: true
+      img_src: /images/nav-icons/Youtube.svg
 - name: Resources
   submenu: true
   highlight: false
   subitems:
+    - name: Cloud Native Catalog
```
Comment (leecalcote) [inline _data/navigation.yml:63]: We will probably have a merge conflict here…

**Comment by leecalcote** [comment]
Thank you for the screenshots. It’s hard to tell, but are we sure that the icon colors are the same as a hyperlink colors?

**Comment by Rohith-Raju** [comment]
@leecalcote this is how it looks on mobile. Should I remove/hide the icons

**Comment by Rohith-Raju** [comment]
@leecalcote Yes its the charcoal color you asked for. I extracted from the menu item hyperlinks itself

File: _data/navigation.yml (hunk 1 above)
Comment (Rohith-Raju) [reply _data/navigation.yml:63]: Oh ok, what can be done @leecalcote

**Review by leecalcote — APPROVED** [review]

**Pull request merged**